torch/csrc/autograd/function.cpp

```diff
@@ -30,6 +30,24 @@ auto Function::flags(const variable_list& inputs) -> FunctionFlags {
   return f;
 }
 
+auto Function::flags(const std::initializer_list<tensor_list> &inputs) -> FunctionFlags {
```

Comment:
It's better to implement `Function::flags` in terms of begin and end iterators (this can support both `initializer_list`s and `variable_list`s), and add the overloads that dispatch to it in `function.h` so they can be inlined.